hypothesis_sqlalchemy/tabular/records.py

```diff
@@ -1,6 +1,6 @@
 from typing import Optional
 
-from hypothesis.searchstrategy.collections import TupleStrategy
+from hypothesis.strategies.internal.collections import TupleStrategy
```

Comment:
instead of `TupleStrategy` we can use just `Strategy`

Reply:
Ok, will do :)

Reply:
Uh, I just looked at it and I do not know what you mean.
I can't find any Strategy class. In hints.py there is an alias Strategy = SearchStrategy

You mean to use that alias?